fal/controllers/anime_stats.py

```diff
@@ -140,9 +143,24 @@ def populate_anime_weekly_stats() -> None:
     week = config.getint("weekly info", "current-week")
 
     with session_scope() as session:
-        anime_list = Season.get_season_from_database(
+        season_anime = Season.get_season_from_database(
             season_of_year, year, session).anime
 
+        anime_ids_collected = (row[0] for row in session.query(AnimeWeeklyStat.anime_id).filter(
+            AnimeWeeklyStat.week == week
+        ).all())
+
+        if anime_ids_collected:
+            action = input("At least some anime stats have been collected for this week"\
+                " already. How should we proceed (overwrite/collect-missing/abort)?")
+            if action == 'collect-missing':
+                anime_list = (anime for anime in season_anime if anime.id in anime_ids_collected)
```

Comment:
I might be misunderstanding this, but 'collect-missing' seems to be collecting information on series that haven't been collected already. Shouldn't the expression be:
```python
anime_list = (anime for anime in season_anime if anime.id not in anime_ids_collected)
```
Then the anime list would contain anime that haven't had stats collected for them so that it can be iterated over below to get stats.

Reply:
Good catch